### DISCUSSION TITLE
Landing at VNLK Ideas

### DISCUSSION THREAD

**Discussion by cameroro** [albar965/littlenavmap#1029] [General]
Hi All
I still am having an obsession with landing at Hillary Tenzing Airport (VNLK) runway 06 in the  Aerobask Epic 1000 turbo prop.I am trying to position the way points on the base leg and  right hand final route so that the aircraft lines up directly with the runway.
I am aiming for a landing speed of 100 knots.The aircraft has to stick to the left hand side of the valley (Nakchun) to give it space when turning left onto finals. The aircraft trail is turned on to help reposition the way points but this is an arc not a straight line. I think that the speed  of the aircraft affects the arc radius. Should I put in more way points or as a last resort turn off the Auto Pilot.
I have include a picture of my flight plan and  aircraft trail.
Any help would most appreciated.
 
![HTA](https://github.com/albar965/littlenavmap/assets/42054234/a82ee788-d1f5-4332-a1c8-e24df109aa32)
![VNLK](https://github.com/albar965/littlenavmap/assets/42054234/8a891849-a264-41a6-984c-1f9c83dfbc54)

**Comment by albar965** [comment]
> Hi All I still am having an obsession with landing at Hillary Tenzing Airport (VNLK) runway 06 in the Aerobask Epic 1000 turbo prop.I am trying to position the way points on the base leg and right hand final route so that the aircraft lines up directly with the runway. I am aiming for a landing speed of 100 knots.The aircraft has to stick to the left hand side of the valley (Nakchun) to give it space when turning left onto finals. The aircraft trail is turned on to help reposition the way points but this is an arc not a straight line. I think that the speed of the aircraft affects the arc radius. Should I put in more way points or as a last resort turn off the Auto Pilot. I have include a picture of my flight plan and aircraft trail. Any help would most appreciated.
> 

You can move the waypoints more south so that the AP follows the path. 

But what is the fun letting the autopilot fly the approach? Fly it manually. Touch down as slow as possible with roughly 83 KTS. Approach is 90 to 100 knots. Do not aim for a smooth touchdown but for an early touchdown on or close to the center line.
I think the real pilots do not use an autopilot in these narrow valleys. Look at YT videos to see how they fly it.

Really like the Aerobask EPIC. Currently flying this thing around in Australia. :slightly_smiling_face: 

Alex
